.teamcity/_self/projects/WPComTests.kt

```diff
@@ -301,6 +301,7 @@ fun jetpackAtomicDeploymentE2eBuildType( targetDevice: String, buildUuid: String
 			param("env.VIEWPORT_NAME", "$targetDevice")
 			param("env.JETPACK_TARGET", "wpcom-deployment")
 			param("env.TEST_ON_ATOMIC", "true")
+			param("env.AUTHENTICATE_ACCOUNTS", "jetpackAtomicDefaultUser,jetpackAtomicPhpOldUser,jetpackAtomicPhpNewUser,jetpackAtomicEcommPlanUser,jetpackAtomicPrivateUser,jetpackAtomicWpBetaUser,jetpackAtomicWpPreviousUser")
```

Comment:
Adds all 7 Jetpack AT users to the pre-authenticate account list, to obtain warm cookies.